### PR TITLE
Storefront, api, load, Payment updates

### PR DESCRIPTION
**Pull request by junior**
Storefront:
- security updates: node packages dependencies (no changes on devDependencies)
- image changed from hub/nginx to nginxinc/nginx-unprivileged
- started to apply hardening practices. Using CIS and STIG/NIST
- new favicon using MuCat
- favicons for each platform
- Multi-arch support for linux/amd64,linux/arm64 platforms
- Helm charts updated to use the latest image tag
- Manifests updated

Payment:
- Helm charts updated to use the latest image tag
- Manifests updated
- Port updated to use non-root port (Root Be Gone)
- Probes updated to use new port

api:
- security updates: node packages dependencies (no changes on devDependencies)
- `connect-redis` pending due breaking changes since v4.0.0. Currently using v3.4.2. Target: 5.2.0.
- connect-redis does not pack redis anymore, and need to use the redis module, generating conflict with session
- started to apply hardening practices. Using CIS and STIG/NIST
- Multi-arch support for linux/amd64,linux/arm64 platforms
- Helm charts updated to use the latest image tag
- Manifests updated

Load:
- Multi-arch support for linux/amd64,linux/arm64 platforms
- Manifests updated
- Python updated to 3.9

### CODE REVIEW

File: src/storefront/package.json [hunk 1]
```diff
@@ -12,9 +12,9 @@
   },
   "dependencies": {
     "axios": "^0.21.1",
-    "core-js": "^3.0.1",
-    "echarts": "^4.2.1",
-    "uikit": "3.1.4"
+    "core-js": "^3.12.1",
+    "echarts": "^5.1.1",
+    "uikit": "3.6.21"
```
Comment (mvandervliet) [inline src/storefront/package.json:17]: Has this bump been tested e2e? `uikit` had been locked to 3.1.4 due to a prior breaking change. Haven't reviewed latest...

Comment (junior) [reply src/storefront/package.json:17]: I've tested manually with the test image and production image. Follow the whole process and didn't found any issues. Using Api Mock and Standard.

Comment (junior) [reply src/storefront/package.json:17]: Do you know what was breaking? Suggests any other tests?

Comment (mvandervliet) [reply src/storefront/package.json:17]: I seem to recall there was a problem with the switcher used in login/register
image

Comment (junior) [reply src/storefront/package.json:17]: @mvandervliet, This was using `mushop/mushop-storefront:2.2.0-RC0`. Does not see issues. Tested with local Kubernetes.

mushop-storefront-RC0

Currently the only security update that was not able to include on this PR, was on the API, for the `connect-redis` that has a "high risk vulnerability". Currently using v3.4.2. Updating to v4, redis is not pre-packaged, and need to change the code. I've changed everything, but shows connect error to the mushop-session, as appear to create incompatibility. I've rollbacked the change and will get back in the future. Will probably need to update mushop-session first.

**Review by mvandervliet — APPROVED** [review]

**Pull request merged**